flink-connector-aws/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/internals/publisher/fanout/FanOutRecordPublisherConfiguration.java

```diff
@@ -472,4 +483,18 @@ public Optional<String> getConsumerName() {
     public Optional<String> getStreamConsumerArn(String stream) {
         return Optional.ofNullable(streamConsumerArns.get(stream));
     }
+
+    public RecoverableErrorsConfig parseRecoverableErrorConfig(String json) {
+        try {
+            return RecoverableErrorsConfig.fromJson(json);
+        } catch (JsonProcessingException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    /** Return recoverable errors config. */
```

Comment:
Remove unnecessary comments https://flink.apache.org/how-to-contribute/code-style-and-quality-common/#comments